### PR TITLE
[WIP] Add batch import test

### PR DESCRIPTION
**Pull request by stremlenye**

### CODE REVIEW

File: features/import.feature [hunk 1]
```diff
@@ -138,3 +138,64 @@ Feature: Import products
     """
     [ '[row 0] Product deleted.' ]
     """
+
+    Scenario: Batch import
+
+      Given a file named "i.csv" with:
+      """
+      id,productType,slug.en,variantId,name,sku,attr-text-n
+      0912,ImpEx with all types,slug_1,1,myProduct,1,key_1
```
Comment (hajoeichler) [inline features/import.feature]: The same id for all products does not make sense here.
Simply remove the column.

File: features/import.feature [hunk 2]
```diff
@@ -138,3 +138,64 @@ Feature: Import products
...
+      0912,ImpEx with all types,slug_4,1,myProduct,4,key_1
+      0912,ImpEx with all types,slug_5,1,myProduct,5,key_1
+      0912,ImpEx with all types,slug_6,1,myProduct,6,key_1
+      0912,ImpEx with all types,slug_7,1,myProduct,7,key_1
+      0912,ImpEx with all types,slug_8,1,myProduct,8,key_1
+      0912,ImpEx with all types,slug_9,1,myProduct,9,key_1
+      0912,ImpEx with all types,slug_10,1,myProduct,10,key_1
+      0912,ImpEx with all types,slug_11,1,myProduct,11,key_1
+      0912,ImpEx with all types,slug_12,1,myProduct,12,key_1
+      0912,ImpEx with all types,slug_13,1,myProduct,13,key_1
```
Comment (hajoeichler) [inline features/import.feature]: Let's add some variants please.

Comment (stremlenye) [inline features/import.feature]: Ok

**Comment by hajoeichler** [comment]
:+1:

**Pull request merged**